test/Confluent.Kafka.IntegrationTests/Tests/AssignPastEnd.cs

```diff
@@ -23,12 +23,12 @@
 
 namespace Confluent.Kafka.IntegrationTests
 {
-    /// <summary>
-    ///     Test functionality of Consumer.Consume when assigned to offest
-    ///     higher than the offset of the last message on a partition.
-    /// </summary>
     public static partial class Tests
     {
+        /// <summary>
+        ///     Test functionality of Consumer.Consume when assigned to offest
+        ///     higher than the offset of the last message on a partition.
+        /// </summary>
```

Comment:
Not really, the application needs to decide what should happen if a sought offset doesn't exist. The client helps with this by allowing the application to select the three possible alternatives: start from beginning, start from end, error out